### DISCUSSION TITLE
new feature Vorgesetzter when creating a user

### DISCUSSION THREAD

**Discussion by bigvictorio** [kimai/kimai#4262] [General] [answered]
Hello,

today when I was creating a new user I noticed a new column "**Vorgesetzter**" (in English: Supervisor) but I could not find any info about this in the changelogs/documentation/forum.

I know it's probably too early for some documentation, but do we have some information on what that feature is good for, maybe some use-case, and what exactly it is supposed to do?

Thanks 🧐

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/documentation/users.html#supervisor
